facefusion/uis/components/webcam.py

```diff
@@ -27,15 +27,17 @@
 WEBCAM_START_BUTTON : Optional[gradio.Button] = None
 WEBCAM_STOP_BUTTON : Optional[gradio.Button] = None
 
-
-def get_webcam_capture() -> Optional[cv2.VideoCapture]:
+def get_webcam_capture(webcam_mode: WebcamMode = None) -> Optional[cv2.VideoCapture]:
 	global WEBCAM_CAPTURE
-
+	webcam_mode = get_ui_component('webcam_mode_radio')
```

Comment:
this ui component register and get stuff is for the communication between different component files. you do not need this

1. the webcam mode is passed via the method
2. you can access the global Gradio element viw WEBCAM_URL_TEXTBOX within this component